src/ORM/DataObjectSchema.php

```diff
@@ -1242,4 +1242,50 @@ protected function checkRelationClass($class, $component, $relationClass, $type)
             );
         }
     }
+
+    /**
+     * @param string|object $baseClass
+     * @param bool $includeBaseClass Whether to include fields in the base class or not
+     * @param string|array $fieldNames The field to get mappings for, for example 'HTMLText'. Can also be an array.
+     * @return array Returns an array of the fields available for the provided class and its sub-classes as follows:
```

Comment:
That looks like something that would be more suitable for a method description than a `@return` description